src/controllers/album.js

```diff
@@ -0,0 +1,116 @@
+//src/controllers/album.js
```

Comment:
Adding the name of the file at the top like this is redundant and should be taken out. This happens in quite a few places throughout the project and all should be removed